### ISSUE TITLE
FR: sanitize folder creation in manager

### ISSUE THREAD

**Issue by pawelmil** [Sterc/FileSluggy#7]
Is it possible to sanitize folder names during it's creation or renaming? Clients often make strange folder names.... Thanks in advance.

**Comment by Bumme** [comment]
Yes that's possible but only with the newest FileSluggy and MODX 2.3

**Comment by pawelmil** [comment]
Can You tell how to configure it? I can't see a system setting for this feature...
